src/Database/Schema/CollectionInterface.php

```diff
@@ -26,7 +26,7 @@ interface CollectionInterface
 {
     /**
      * Get the list of tables available in the current connection.
-     *
+     * @deprecated
```

Comment:
This interface could use the method annotations as we can't add the methods here.